### PR TITLE
[#176738799] Add tests resources in to UAT PROD payment manager env

### PR DESCRIPTION
**Pull request by pasqualespica**
#### Description 

The goal of this PR is to tests `/resources` endpoint in to UAT and PROD PM environment

#### List of Changes

- modify `.env` file with new info for UAT and PROD PM env
- add integration tests `vs` UAT and PROD PM env, related to `v4/resources`
- add `jest` config file for UAT and PROD PM env
- modify `package.json` to extend `yarn test` behavior to include UAT/PROD test

#### Motivation and Context

The invocation of the following api (Payment Manager):

  - GET `pp-restapi/v4/resources?language=it`

#### How Has This Been Tested?

- `yarn test:uat-prod-int`
> NOTE : before typing `yarn install && yarn generate`

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/__integrations__/uat-prod/pmClient.resources.integration.uat-prod.test.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      baseUrl: `https://${PM_HOST_UAT}:${PM_PORT}/pp-restapi` as string,
+      fetchApi: retryingFetch(fetch, 5000 as Millisecond, 5),
+    });
+    const paymentManagerClientPROD = createClient({
+      baseUrl: `https://${PM_HOST_PROD}:${PM_PORT}/pp-restapi` as string,
+      fetchApi: retryingFetch(fetch, 5000 as Millisecond, 5),
+    });
+    // Spy on th global fetch to check if it gets called
+    const mySpyGlobalFetch = jest.spyOn(global, 'fetch');
+    expect(paymentManagerClientUAT).toBeTruthy();
```
Comment (infantesimone) [inline src/__integrations__/uat-prod/pmClient.resources.integration.uat-prod.test.ts:51]: Why is this test done in _beforeEach_? What is the goal?

File: src/__integrations__/uat-prod/pmClient.resources.integration.uat-prod.test.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
+// import { fromNullable } from 'fp-ts/lib/Option';
```
Comment (infantesimone) [inline src/__integrations__/uat-prod/pmClient.resources.integration.uat-prod.test.ts:1]: To delete

**Review by infantesimone — APPROVED** [review]

File: src/__integrations__/uat-prod/pmClient.resources.integration.uat-prod.test.ts (hunk 1 above)
Comment (pasqualespica) [reply src/__integrations__/uat-prod/pmClient.resources.integration.uat-prod.test.ts:51]: Tnx for pointing out it, it isn't useful, it is the result of a copy and paste from another test. I'll remove it.

**Review by GiovanniMancini — APPROVED** [review]
Thank you. Theese tests are a great added value for the project since they introduce a way to automate tests in production!

**Pull request merged**